components/Home/Donate/index.jsx

```diff
@@ -75,6 +85,12 @@ export const DepositServiceDonation = () => {
             <a href="https://member.olas.network/" target="_self">
               here
             </a>
```

Comment:
Text above should change:
"Service owners can boost incentives of devs who contributed code to their services with freshly minted OLAS, by owning" -> "To boost incentives of devs with freshly minted OLAS, you must hold at least"